### PR TITLE
Add time measurement for each step and whole run

### PR DESCRIPTION
**Pull request by eugeneniemand**

### CODE REVIEW

File: psmake/psmake.ps1 [hunk 1]
```diff
@@ -326,4 +331,6 @@ catch [Exception]
 finally
 {
 	if($AnsiConsole) { remove-item function:Write-Host }
+	$overall_sw.Stop()    
+	Write-ShortStatus "PsMake run duration: $($overall_sw.Elapsed)"
```
Comment (Suremaker) [inline psmake/psmake.ps1]: that change has to happen before if($AnsiConsole)

File: psmake/psmake.ps1 [hunk 2]
```diff
@@ -283,17 +283,21 @@ function private:Execute-Steps([array]$steps)
 	{
 		if (Should-RunStep ($steps[$i].Name)) {
 			Write-Header -style "*" -header "$($i+1)/$($steps.Length): $($steps[$i].Name)..."
-			& ($steps[$i].Body) 
+			$sw = [Diagnostics.Stopwatch]::StartNew()
+			& ($steps[$i].Body) 			 
 			if (-not $?) { throw 'Last step terminated with error...' }
+			$sw.Stop()    
+        	Write-ShortStatus "$($steps[$i].Name) run duration: $($sw.Elapsed)"
```
Comment (Suremaker) [inline psmake/psmake.ps1]: Text indentation is incorrect

File: psmake/ChangeLog.txt [hunk 3]
```diff
@@ -1,5 +1,8 @@
 PsMake
 ========================================
+Version 3.1.7.0
+----------------------------------------
+General: Added run duration for Overall script and for each step.
 
 Version 3.1.6.0
```
Comment (Suremaker) [inline psmake/ChangeLog.txt]: All the time related new changes can be put to 3.1.6. I will release all the changes in one go

File: psmake/psmake.ps1 (hunk 2 above)
Comment (eugeneniemand) [inline psmake/psmake.ps1]: Think its a tab vs space issue. I think its fixed now

File: psmake/psmake.ps1 (hunk 1 above)
Comment (eugeneniemand) [inline psmake/psmake.ps1]: Done

**Pull request merged**